dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -576,6 +579,18 @@ function inputsToDict(inputs_list: any) {
     return inputs;
 }
 
+function getTriggeredId(triggered: string[]): string | object {
```

Comment:
The function needs to return something or add `string | object | undefined` to the return type.

Reply:
It's a renderer build error in install-dependencies-36:
```
ERROR in /home/circleci/dash/dash/dash-renderer/src/actions/callbacks.ts
./src/actions/callbacks.ts 582:46-61
[tsl] ERROR in /home/circleci/dash/dash/dash-renderer/src/actions/callbacks.ts(582,47)
      TS2366: Function lacks ending return statement and return type does not include 'undefined'.
ts-loader-default_e3b0c44298fc1c14
 @ ./src/actions/index.js 6:0-52 72:11-32 111:21-42
 @ ./src/AppContainer.react.js 24:0-54 40:21-29 58:15-24
 @ ./src/AppProvider.react.tsx 11:0-48 21:38-50
 @ ./src/DashRenderer.js 8:0-46 15:50-61 19:54-65
 @ ./src/index.js 1:0-46 4:22-34

webpack 5.88.2 compiled with 1 error in 22846 ms
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dash-renderer@1.17.0 build:js: `webpack`
npm ERR! Exit status 1
npm ERR! 
```

Reply:
I see, thanks. That's pretty buried - the step the actual error occurred in appears to succeed, then the step afterward fails cryptically. Worse, install-dependencies-39 has the same error but doesn't appear to fail at all (the logs moved?), so we try to run all the test jobs for Py3.9 and nearly everything fails, making it even harder to find the root cause.

Anything we can do to fix this, ie make the "Build Component Packages & Update Dependencies/Artifacts" step itself fail rather than relying on finding error logs to tell us it failed?